tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+import logging
+
+import pytest
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture(scope="session")
```

Comment:
Consider using https://github.com/holdenk/spark-testing-base

Reply:
I did take a look but felt that the Python implementation doesn't do much more than what's being done with the pytest fixtures. If later we think it's beneficial we can always switch over.